### PR TITLE
mask in loss function did not have the correct shape

### PR DESCRIPTION
**Pull request by borundev**
The loss_function in master is

  mask = tf.math.logical_not(tf.math.equal(real, 0))
......
  loss_ *= mask

  but mask and loss have different shapes so it is not doing the intended action of masking away the loss when real values are 0 and instead leads to a broadcasted matrix.

The second line should instead be

loss_ *= tf.reshape(mask,(-1,))

PS. I am not quite familiar with how to submit ipynb files with git and it seems the whole file has been changed and is not showing a meaningful diff. If there is some procedure/tool to do it properly please let me know and I will do it.

### CODE REVIEW

**Comment by borundev** [comment]
@googlebot I signed it!

**Comment by borundev** [comment]
@googlebot I fixed it.

**Comment by borundev** [comment]
@googlebot I fixed it.

**Comment by borundev** [comment]
@googlebot I signed it!

**Comment by borundev** [comment]
@googlebot I fixed it

**Comment by borundev** [comment]
@googlebot I fixed it

**Comment by lamberta** [comment]
I verified the CLA, thank you

**Comment by lamberta** [comment]
To make the diffs easier to read, can you please run notebooks through [nbfmt.py](https://github.com/tensorflow/docs/blob/master/tools/nbfmt.py) ?
Thanks

**Comment by borundev** [comment]
@lamberta I ran nbfmt on the notebook. it introduced some extra changes on top (like replacing an encoding of < with < itself). Presumably that's an intended behavior. Had me confused for a whole. My change is limited to line 676 of the master version.

Also googlebot changed my CLA back to no. Since you manually changed it to yes last time, can you let me know if there is something I can do to not have the bot do this again?

**Comment by borundev** [comment]
I am closing the pull request as I think it is not a but in the context in which it was being used. I seem to have seen the error when I was experimenting with it and passing more general rank tensors.
